### PR TITLE
Swift Package Manager support

### PR DESCRIPTION
**Pull request by smamczak**
I have add the Package.swift file and moved NibView.swift into Sources if you don't mind

### CODE REVIEW

**Review by domasn — COMMENTED** [review]
Thanks for showing the initiative! 👍

Though, for `#if TARGET_INTERFACE_BUILDER` to work inside `NibView.swift`, that file needs to be added directly to the project and not pre-compiled. See how [installation guide](https://github.com/trafi/NibView#installation) suggests adding this directly to your project.

I understand, that this is a big limitation (especially for wanting to use it with SPM) for when `@IBDesignable` is not needed, but _the point_ of this library is to add better interface builder support for nibs.

Could you check if `@IBDesignables` work properly (views render inside interface builder as you'd expect) and `@IBInspectable` values are copied when using SPM?

**Comment by smamczak** [comment]
Hi there. Maybe something changed but in the newest Xcode 11.4.1  `#if TARGET_INTERFACE_BUILDER` works good from the SPM package.

I have prepared a demo for you. I've removed Sources and added SPM dependency to `spm` branch. You can download it and see for your self

https://github.com/smamczak/NibView/tree/spm-with-example

**Comment by domasn** [comment]
This is great news! 🎉 
Then, I suggest we keep the original `NibView.swift` file as it was (outside of sources - for Carthage compatibility) and add a copy of it under Sources for SPM to work like magic.

**Comment by smamczak** [comment]
I've managed to configure Packages.swift without copying the NibView.swift

File: README.md [hunk 1]
```diff
@@ -94,5 +94,14 @@ Drag [`NibView.swift`](NibView.swift) file into your Xcode project and add the f
...
 ```
+
+## [Swift Package Manager](https://swift.org/package-manager/)
+```
+dependencies: [
+    .package(url: "https://github.com/trafi/NibView.git", .upToNextMajor(from: "2.0.0"))
+]
+```
+
+## []
```
Comment (domasn) [inline README.md]: Seems like a leftover.
```suggestion
```

File: NibView.swift [hunk 2]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2016 Trafi. All rights reserved.
 //
 
-import NibView
+import UIKit
```
Comment (domasn) [inline NibView.swift:11]: We still need it for Carthage installation:
```suggestion
#if canImport(NibView)
import NibView #endif
import UIKit
```

File: README.md [hunk 3]
```diff
@@ -94,5 +94,14 @@ Drag [`NibView.swift`](NibView.swift) file into your Xcode project and add the f
 ```
 github "Trafi/NibView"
 ```
+
+## [Swift Package Manager](https://swift.org/package-manager/)
```
Comment (domasn) [inline README.md:98]: 😍

**Review by domasn — APPROVED** [review]

**Comment by domasn** [comment]
Thanks for contributing 🙇‍♂️
I've created a [new release](https://github.com/trafi/NibView/releases/tag/2.0.1) so SPM finds `Package.swift` in it.

**Comment by smamczak** [comment]
Thank you:)

**Pull request merged**